apps/src/applab/commands.js

```diff
@@ -1140,6 +1140,27 @@ applabCommands.setProperty = function (opts) {
   Applab.updateProperty(element, info.internalName, value);
 };
 
+applabCommands.getProperty = function (opts) {
+  apiValidateDomIdExistence(opts, 'getProperty', 'id', opts.elementId, true);
+  apiValidateType(opts, 'getProperty', 'property', opts.property, 'string');
+
+  var elementId = opts.elementId;
+  var property = opts.property;
+
+  var element = document.getElementById(elementId);
+  if (!element) {
+    return;
+  }
+
+  var info = setPropertyDropdown.getInternalPropertyInfo(element, property);
+  if (!info) {
+    outputError('Cannot get property "' + property + '" on element "' + elementId + '".');
+    return;
+  }
+
+  return Applab.readProperty(element, info.internalName);
```

Comment:
could be const instead of var in this whole block. Could also use string templating when generating your error string.